File: cookbook_flutter/lib/Recipe.dart
import 'package:flutter/foundation.dart';
import 'BodySegment.dart';

class Recipe {
  String id;
  String title;
  String description;
  String author;
  List ingredients;
  int servings;
  List<BodySegment> body;

  Recipe(
      {@required this.id,
      @required this.title,
      @required this.description,
      @required this.author,
      @required this.ingredients,
      @required this.servings,
      @required this.body});
}

File: cookbook-desktop/src/Models/Recipe.js
class RecipeSchema {
  constructor({
    _id,
    title,
    description,
    author,
    ingredients,
    servings,
    body,
  }) {
    this._id = _id;
    this.title = title;
    this.description = description;
    this.author = author;
    this.ingredients = ingredients;
    this.servings = servings;
    this.body = body;
  }
}

export default RecipeSchema;
